src/utils/types.js

```diff
@@ -168,3 +168,80 @@ export const createSourceWithParamsPropType = {
   id: PropTypes.string,
   last4: PropTypes.string,
 }
+
+
+// Corresponds to https://stripe.com/docs/api/payment_methods/create
+export const createPaymentMethodPropType = {
+
+  id: PropTypes.string,
+
+  // BillingDetails properties:
+  billingDetails: PropTypes.shape({
+    address: PropTypes.shape({
+      city: PropTypes.string,
+      country: PropTypes.string,
+      line1: PropTypes.string,
+      line2: PropTypes.string,
+      postalCode: PropTypes.string,
+      state: PropTypes.string,
```

Comment:
Should this be "state/province"? (Thinking internationally)

Reply:
we're using the same terms as Stripe for parity with their API (this isn't something we would modify here)